starknet_py/tests/e2e/docs/guide/test_sign_offchain_message.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+
+
+@pytest.mark.asyncio
+async def test_sign_offchain_message():
+    # pylint: disable=import-outside-toplevel, duplicate-code, unused-variable
+
+    # add to docs: start
+    from starknet_py.net import AccountClient, KeyPair
+    from starknet_py.net.gateway_client import GatewayClient
+    from starknet_py.net.models import StarknetChainId
+    from starknet_py.net.models.typed_data import Parameter, StarkNetDomain, TypedData
+
+    # Create components of TypedData TypedDict
+    starknet_domain = StarkNetDomain(name="StarkNet Mail", version="1", chainId=1)
+
+    types = {
+        "StarkNetDomain": [
+            Parameter(name="name", type="felt"),
+            Parameter(name="version", type="felt"),
+            Parameter(name="chainId", type="felt"),
+        ],
+        "Person": [
+            Parameter(name="name", type="felt"),
+            Parameter(name="wallet", type="felt"),
+        ],
+        "Mail": [
+            Parameter(name="from", type="Person"),
+            Parameter(name="to", type="Person"),
+            Parameter(name="contents", type="felt"),
+        ],
+    }
+
+    message = {
+        "from": {"name": "Cow", "wallet": "0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826"},
+        "to": {"name": "Bob", "wallet": "0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB"},
+        "contents": "Hello, Bob!",
+    }
+
+    # Create a TypedData dictionary
+    typed_data = TypedData(
+        types=types,
+        primaryType="Mail",
+        domain=starknet_domain,
+        message=message,
+    )
```

Comment:
Could we replace these steps with the creation of a single standard dict to show it is possible to use `sign_message` without explicitly instantiating TypedData typeddict?

Something like
```python
typed_data = {
  "types": {...},
  "primary_type": "Mail"
  "domain": {...},
  "message: {...}
}
```

Current approach suggests it's necessary to create all these objects.